client/src/pages/Project.js

```diff
@@ -121,19 +163,6 @@ class Project extends React.Component {
       this.props.history.push(`/messages/${user.id}`);
     };
 
-    const handleFundProject = () => {
-      const { history, userDetails, match } = this.props
-
-      history.push({
-        pathname: '/checkout',
-        state: {
-          userId: userDetails.id,
-          projectId: match.params.id,
-          projectTitle: this.state.project.title
-        }
-      })
-    };
-
     const disableFunding = () => {
       // TODO: Logic to handle disabling funding of project, maybe if project fund period has ended
       // Disabled by default for now until funding logic is added
```

Comment:
Can remove todo

Reply:
You don't think this should still be done?

Reply:
if not, maybe I remove the function itself?